chapters/ch20.md

```diff
@@ -1,49 +1,57 @@
 # 20. Bias and Variance: The two big sources of error
 ->
-
+# 20. Độ chệch và phương sai: Hai nguồn lớn của lỗi
 
 Suppose your training, dev and test sets all come from the same distribution. Then you should always try to get more training data, since that can only improve performance, right?
 ->
-
+Giả sử khi huấn luyện, tập phát triển và tập kiểm tra có cùng phân phối. Khi đó bạn luôn nên cố gắng thu thập thêm dữ liệu, vì điều đó chỉ giúp cải thiện hiệu quả, đúng không? (cần dịch lại)
 
 Even though having more data can’t hurt, unfortunately it doesn’t always help as much as you might hope. It could be a waste of time to work on getting more data. So, how do you decide when to add data, and when not to bother?
 ->
+Mặc dù có thêm dữ liệu không thể giảm hiệu quả, thật không may điều này không luôn luôn mang lại hiệu quả nhiều như bạn mong đợi. Việc thu thập thêm dữ liệu có thể trở nên lãng phí thời gian. Vậy làm thể nào để quyết định khi nào nên thêm dữ liệu, khi nào không nên?
 
 
 There are two major sources of error in machine learning: bias and variance. Understanding them will help you decide whether adding data, as well as other tactics to improve performance, are a good use of time.
 ->
-
+Có hai nguồn chính dẫn đến lỗi trong machine learing: độ chệch và phương sai. Hiểu được chúng sẽ giúp bạn quyết định liệu rằng có đáng bỏ ra thời gian để thêm dữ liệu, cũng như các kỹ thuật khác để cải thiện hiệu quả của mô hình.
 
 Suppose you hope to build a cat recognizer that has 5% error. Right now, your training set has an error rate of 15%, and your dev set has an error rate of 16%. In this case, adding training data probably won’t help much. You should focus on other changes. Indeed, adding more examples to your training set only makes it harder for your algorithm to do well on the training set. (We explain why in a later chapter.)
 ->
+Giả sử bạn hy vọng xây dựng một bộ nhận dạng mèo với 5% lỗi. Hiện tại, tập huấn luyện của bạn có tỉ lệ lỗi là 15%, và tập phát triển có tỉ lệ lỗi là 16%. Trong trường hợp này, việc thêm dữ liệu có thể không giúp được gì nhiều. Bạn nên tập trung vào các thay đổi khác. Chắc chắn rằng tăng lượng mẫu cho tập huấn luyện chỉ khiến thuật toán của bạn làm việc tốt trên tập mẫu này trở nên khó khăn hơn. (Chúng tôi sẽ giải thích trong một chương sau.)
```

Comment:
" Chắc chắn rằng tăng lượng mẫu cho tập huấn luyện chỉ khiến thuật toán của bạn làm việc tốt trên tập mẫu này trở nên khó khăn hơn" ==> " Chắc chắn rằng tăng lượng mẫu cho tập huấn luyện chỉ khiến thuật toán của bạn khó học trên tập mẫu hơn, khó học được một mô hình tốt hơn mô hình hiện tại" (câu này khó dịch thoát ý quá)

Reply:
Mình nghĩ mình và bạn đều hiểu đúng câu này, chỉ là chưa tìm được cách dịch hay hơn.

Reply:
" Chắc chắn rằng tăng số lượng mẫu cho tập huấn luyện chỉ khiến việc huấn luyện mô hình của bạn trên đó trở nên khó hơn"? (ở đây assume là your algorithm = your model).

Reply:
@william-vu Em sửa theo bác, thêm chữ **tập** vào trước chữ "đó" nữa là ổn.